apps/native/app/src/screens/inbox/inbox.tsx

```diff
@@ -202,34 +205,36 @@ function useInboxQuery(incomingFilters?: Filters) {
 
   useEffect(() => {
     const numItems = data?.totalCount ?? pageSize
+
     if (pageSize * (page - 1) > numItems) {
       return
     }
+
     setLoading(true)
+
     // Fetch data
-    client
-      .query<ListDocumentsQuery, ListDocumentsQueryVariables>({
-        query: ListDocumentsDocument,
-        fetchPolicy: 'network-only',
-        variables: {
-          input: {
-            page,
-            pageSize,
-            ...filters,
-          },
+    getListDocument({
+      variables: {
+        input: {
+          page,
+          pageSize,
+          ...filters,
         },
-      })
-      .then((res) => {
-        if (page > 1) {
-          setData((prevData) => ({
-            data: [
-              ...(prevData?.data ?? []),
-              ...(res.data.listDocumentsV2?.data ?? []),
-            ],
-            totalCount: res.data.listDocumentsV2?.totalCount,
-          }))
-        } else {
-          setData(res.data.listDocumentsV2)
+      },
+    })
+      .then(({ data }) => {
+        if (data) {
```

Comment:
~Can `data` be null here?

Reply:
Yep 😢  
![image](https://github.com/island-is/island.is/assets/112904566/1e33f26b-5119-40e2-9a8b-a73e7b815b34)